src/IndexedDB/Browser.IndexedDB.fs

```diff
@@ -0,0 +1,111 @@
+﻿namespace rec Browser.Types
+
+open System
+open Fable.Core
+
+type DatabasesType = {
+    name: string
+    version: string
+}
```

Comment:
It's better to use always interfaces for the bindings. This is because F# records have properties (like structural equality) that are not usually present in JS objects.

> There's an anonymous record in Browser.Dom, but I added it by mistake and I cannot remove it now to avoid a breaking change.

Reply:
I'll give it a try! :)